### ISSUE TITLE
Redesign the "UseCase" Card on Android

### ISSUE THREAD

**Issue by anku255** [MLH-Fellowship/react-native#134]
The use case card in RNTester needs a redesign.

- Use the Material Design Card
- Make title font bigger
- Improve spacing

**Comment by anku255** [comment]
@stealthanthrax Hey! Ricky mentioned that the components should feel like a native platform component. Would you be able to design the use case card for both the platforms?

Here are some links that you may fine useful - 

https://material.io/components/cards
https://setproduct.com/patterns/ios/cards

**Comment by stealthanthrax** [comment]
Sure @anku255 ! Thanks for the links 🙌 . Are you taking on this issue in the current sprint?

**Comment by anku255** [comment]
@stealthanthrax Yes, if you can get the designs ready then I would like to complete it in this sprint as well.

**Comment by stealthanthrax** [comment]
@anku255 , can you take up a different component? I was thinking of tackling the redesign on Friday.😅

**Comment by anku255** [comment]
@stealthanthrax Sure! Do it when you have time.

**Comment by YashKumarVerma** [comment]
@stealthanthrax if you could give a :green_circle: from your side, meanwhile @anku255  is busy on other parts, i can work on it.

**Comment by stealthanthrax** [comment]
Yes @YashKumarVerma , go ahead 👍

**Comment by YashKumarVerma** [comment]
The designs have been published at https://www.figma.com/file/UZ0WPWEuQfr7dwKvirUTAt/Example-Card-Samples?node-id=0%3A1 and are implemented in the accompanied PR.
